artichoke-backend/src/extn/core/time/mod.rs

```diff
@@ -14,11 +14,11 @@
 //! ```
 //!
 //! This implementation of `Time` supports the system clock via the
-//! [`chrono`] and [`chrono-tz`] crates.
+//! [`tz-rs`] and [`tzdb`] crates.
```

Comment:
I wonder if just a reference to `spinoso-time` itself is better than the packages that `spinoso-time` relies on. But just a minor nit /shrug